app/controllers/api/container_images_controller.rb

```diff
@@ -1,4 +1,13 @@
 module Api
   class ContainerImagesController < BaseController
+    def scan_result_summary_resource(type, id = nil, _data = nil)
+      raise BadRequestError, "Must specify an id for starting a #{type} resource" unless id
```

Comment:
starting word need to reflect action name.

for me, scan_result_summary seems ambiguous, I wonder if it should reflect what is being gathered, i.e. something like  "openscap_scan_results" or just "openscap_rule_results" since that's what we're fetching.

Reply:
@abellotti I might need later to have an endpoint that will return the html or xml report that we also have, so I called this summary.
Can we call this `openscap_scan_results` and a probably future endpoint will be `openscap_scan_results_html` ?